common/common_functions.sh

```diff
@@ -126,6 +126,10 @@ get_distro_version(){
     grep ^VERSION_ID= /etc/os-release | cut -d'=' -f2 -s | tr -d '"'
 }
 
+get_kind_distro_version(){
```

Comment:
There is the same function in other PR https://github.com/Mellanox/kubernetes-ci/pull/124/commits/b55619f71493b4e4b287be0ce743c5cf361ab7c6#diff-2ce0144df97c1e95fc4cf476a871acf44e86477b406d5cfd4daab12e32d638c8. will'll probable have a merge conflict here

Reply:
This happens because this PR depends on the other patch. So this patch will need to be rebased after the other patch merged.

Reply:
the last two commits of this PR are the ones that have the functionality wanted.